tfx/examples/imdb/imdb_utils_native_keras.py

```diff
@@ -118,18 +124,18 @@ def _build_keras_model() -> tf.keras.Model:
   # The model below is built with Functional API, please refer to
   # https://www.tensorflow.org/guide/keras/overview for all API options.
   model = tf.keras.Sequential([
-      tf.keras.layers.Embedding(_MAX_FEATURES+1, 64),
+      tf.keras.layers.Embedding(_MAX_FEATURES+1, _HIDDEN_UNITS),
       tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(
-          64,
-          dropout=0.2,
-          recurrent_dropout=0.2)),
-      tf.keras.layers.Dense(64, activation='relu'),
+          _HIDDEN_UNITS,
+          dropout=_DROPOUT_RATE,
+          recurrent_dropout=_DROPOUT_RATE)),
+      tf.keras.layers.Dense(_HIDDEN_UNITS, activation='relu'),
       tf.keras.layers.Dense(1)
   ])
 
   model.compile(
       loss='binary_crossentropy',
-      optimizer=tf.keras.optimizers.Adam(1e-4),
+      optimizer=tf.keras.optimizers.Adam(_LEARNING_RATE),
       metrics=['accuracy'])
 
   model.summary()
```

Comment:
To align _get_serve_tf_examples_fn with the padded batch, after we have popped the label key in transformed_features, place transformed_features within a dataset by calling tf.data.Dataset.from_tensor_slices. Then call padded_batch similarly to how we talked about above, but with batch_size set to tf.shape(serialized_tf_examples)[0]. Finally, call tf.data.experimental.get_single_element.